src/main/java/uk/gov/hmcts/darts/arm/service/impl/DetsToArmBatchPushProcessorImpl.java

```diff
@@ -117,7 +113,8 @@ private List<Integer> getDetsEodEntitiesToSendToArm(ExternalLocationTypeEntity s
     }
 
     private void createAndSendBatchFile(List<ExternalObjectDirectoryEntity> eodsForBatch, UserAccountEntity userAccount) {
-        File archiveRecordsFile = dataStoreToArmHelper.createEmptyArchiveRecordsFile(getManifestFilePrefix());
+        String archiveRecordsFileName = dataStoreToArmHelper.getArchiveRecordsFileName(armDataManagementConfiguration.getManifestFilePrefix());
```

Comment:
This needs to use getManifestPrefix as this version should return DETS but the armDataManagementConfiguration will return DARTS

Reply:
We are using getManifestFilePrefix :)

Reply:
Sorry I didn't word it correctly. It needs to call the method getManifestFilePrefix() which returns "DETS" as the manifest name as it uses the DETS configuration detsToArmProcessorConfiguration.getManifestFilePrefix(). armDataManagementConfiguration.getManifestFilePrefix() returns "DARTS"

Reply:
Aha yes very good spot :)

I have updated it to use the dets config instead of the arm config :)